core/src/test/scala/co.blocke.scalajack/v4/ParseOrElseTests.scala

```diff
@@ -7,26 +7,26 @@ import scala.language.postfixOps
 import scala.util.Try
 import org.joda.time.{ DateTime, DateTimeZone }
 import org.joda.time.format.DateTimeFormat
+import scala.reflect.runtime.universe.{ Type, typeOf }
 
 case class DefaultBlah() extends Blah
 
 class ParseOrElseTests extends FunSpec {
 
-  val sjJS = ScalaJack()
-  val vc = VisitorContext().copy(parseOrElse = Map("co.blocke.scalajack.test.v4.Blah" -> "co.blocke.scalajack.test.v4.DefaultBlah"))
+  val sjJS = ScalaJack().parseOrElse((typeOf[Blah] -> typeOf[DefaultBlah]))
```

Comment:
I thought you might like this. :-)  I saw how you were basically building the Type anyway, so why not make the code simpler and more flexible.  A test case shows that you can still do textual mutation on a fully qualified name and materialize the Type using fullNameToType if you want to, so we didn't lose any capability--just something to clearly document.